addons/payment_nuvei/tests/test_payment_transaction.py

```diff
@@ -0,0 +1,146 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from unittest.mock import patch
+
+from werkzeug import urls
+
+from odoo.tests import tagged
+from odoo.tools import mute_logger
+
+from odoo.addons.payment import utils as payment_utils
+
+from odoo.addons.payment_nuvei.controllers.main import NuveiController
+from odoo.addons.payment_nuvei.tests.common import NuveiCommon
+
+
+@tagged('post_install', '-at_install')
+class TestPaymentTransaction(NuveiCommon):
+
+    def _get_expected_values(self, tx):
```

Comment:
As this is used only once, to test the rendering values, might be better to not extract it and instead let it inside the `test_no_item_missing_from_rendering_values` for clarity